env.local

```diff
@@ -0,0 +1,8 @@
+# Configurações do Firebase
```

Comment:
Firebase config should be in constants, it's fine for security as they can access it anyway

Reply:
The idea was to put env.local in git ignore so it wouldn't appear here.